src/main/java/com/ensao/gi5/lint/rules/BooleanExpressionRule.java

```diff
@@ -0,0 +1,41 @@
+package com.ensao.gi5.lint.rules;
+
+import com.ensao.gi5.lint.constantes.Constantes;
+import com.ensao.gi5.lint.enumeration.Level;
+import com.ensao.gi5.lint.visitor.BooleanExpressionVisitor;
+import com.ensao.gi5.lint.wrapper.BooleanExpressionWrapper;
+import com.ensao.gi5.lint.wrapper.CompilationUnitWrapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BooleanExpressionRule extends Rule {
+
+    public BooleanExpressionRule() {
+        super(Constantes.LINT_REG_006, Level.HIGHEST);
+    }
+
+    @Override
+    public void apply(CompilationUnitWrapper compilationUnit) {
+        List<BooleanExpressionWrapper> booleanExpressions = new ArrayList<>();
+        compilationUnit.accept(new BooleanExpressionVisitor(), booleanExpressions);
+        for (BooleanExpressionWrapper booleanExpression: booleanExpressions) {
+            if (checkIfBooleanExpressionHasMoreThanTwoOperands(booleanExpression)) {
```

Comment:
`checkIfBooleanExpressionHasMoreThanTwoOperands` ==> `booleanExpressionHasMoreThanTwoOperands`
in the reading it will be ` if boolean expression has more than two operands`